japicmp incompatible exception while compilation.

we have added a new method in org.apache.flink.streaming.api.operators.StreamOperator.

When compiling the flink-streaming-java, it returns a exception:
[ERROR] Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.11.0:cmp (default) on project flink-streaming-java_2.11: Breaking the build because there is at least one incompatibility: org.apache.flink.streaming.api.operators.StreamOperator.updateOutput(org.apache.flink.streaming.runtime.tasks.StreamTask,org.apache.flink.streaming.api.operators.Output):METHOD_ADDED_TO_INTERFACE -> [Help 1]

--- a/flink-streaming-java/pom.xml
+++ b/flink-streaming-java/pom.xml
@@ -95,10 +95,10 @@ under the License.
 		<plugins>
 
 			<!-- activate API compatibility checks -->
-<!--			<plugin>-->
-<!--				<groupId>com.github.siom79.japicmp</groupId>-->
-<!--				<artifactId>japicmp-maven-plugin</artifactId>-->
-<!--			</plugin>-->
+			<plugin>
+				<groupId>com.github.siom79.japicmp</groupId>
+				<artifactId>japicmp-maven-plugin</artifactId>
+			</plugin>
 
 			<!-- disable fork reuse for the streaming project, because of
 			incorrect declaration of tests -->

--- a/pom.xml
+++ b/pom.xml
@@ -1896,6 +1896,7 @@ under the License.
 								<exclude>org.apache.flink.streaming.api.environment.LegacyLocalStreamEnvironment</exclude>
 								<exclude>org.apache.flink.streaming.api.functions.sink.RichSinkFunction#invoke(java.lang.Object)</exclude>
 								<exclude>org.apache.flink.streaming.api.functions.sink.SinkFunction</exclude>
+								<exclude>org.apache.flink.streaming.api.operators.StreamOperator</exclude>
 								<exclude>org.apache.flink.api.java.hadoop.mapred.HadoopInputFormat</exclude>
 								<exclude>org.apache.flink.api.java.hadoop.mapred.HadoopOutputFormat</exclude>
 								<exclude>org.apache.flink.api.java.hadoop.mapreduce.HadoopInputFormat</exclude>

